best-practices/README.md

```diff
@@ -187,6 +187,14 @@ Email
 [MailView]: https://github.com/37signals/mail_view
 [ActionMailer Preview]: http://api.rubyonrails.org/v4.1.0/classes/ActionMailer/Base.html#class-ActionMailer::Base-label-Previewing+emails
 
+Web
+---
+
+* Prefer local assets to externally linked assets.
```

Comment:
I think if we're hosting all the static assets on one CDN, we prefer that. However, I think we'd generally prefer using "our" CDN for whatever project it is instead linking to several external CDNs.